pkg/driver/node.go

```diff
@@ -359,3 +379,56 @@ func (d *Driver) nodePublishVolumeForFileSystem(req *csi.NodePublishVolumeReques
 
 	return nil
 }
+
+// findDevicePath finds and verifies the path of device
+// if the device is not nvme device, return the path
+// if the device is nvme device, returns the actual device path
```

Comment:
What does "actual device path" mean vs "the path of device"?

Reply:
clarified the docs